app/new_service_manifest.yml

```diff
@@ -36,7 +36,15 @@
     - educationPricing
     - trialOption
     - freeOption
+-
+  name: Pricing document
```

Comment:
The file upload pages should be the last four pages (to avoid people getting blocked half way through submitting a service). I think the best ordering for them would be: Service definition, T&Cs, Pricing, SFIA rate card.